test/plugin/test_out_google_cloud.rb

```diff
@@ -695,7 +653,7 @@ def test_label_map_with_field_present
     d.run
     # make a deep copy of COMPUTE_PARAMS and add the parsed label.
     params = Marshal.load(Marshal.dump(COMPUTE_PARAMS))
-    params['labels']['sent_label'] = 'label_value'
+    params[:labels]['sent_label'] = 'label_value'
```

Comment:
Unlike the params fields, the label set is a string to string Hash. I didn't want to introduce a symbol to string field, even if it would serialize to JSON correctly.